app/views/obituaries/show.html.erb

```diff
@@ -9,3 +9,8 @@
 <h2> State: <%= @obituary.state %> </h2>
 
 <p><%= @obituary.description %></p>
+
+<% if current_user && current_user.obituaries.include?(@obituary) %>
```

Comment:
Should I make a model method for this? Not sure if it is too much logic in the view.

Reply:
I think a model method would be better. We could put this on a list of refactor tasks.

Reply:
Probably a model method for this one, the include? might be a bit to much for the view

Reply:
My vote is yes since that it is definitely best practice, and it probably won't take that long.